setup.cfg

```diff
@@ -1,3 +1,5 @@
 [flake8]
 max-line-length = 99
-ignore = ""
\ No newline at end of file
+ignore = 
+    W504,  # W504: Line break occurred after a binary operator
+    E741  # E741: Do not use variables named 'I', 'O', or 'l
```

Comment:
Can you add a comment with a TODO pointing to the issue you created, so there is more context of why we ignore this error code?

Reply:
Ye, I'll do it!